osu.Game/Screens/Select/BeatmapClearScoresDialog.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osu.Game.Overlays.Dialog;
+using osu.Game.Scoring;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace osu.Game.Screens.Select
+{
+    public class BeatmapClearScoresDialog : PopupDialog
+    {
+        private ScoreManager manager;
+
+        [BackgroundDependencyLoader]
+        private void load(ScoreManager beatmapManager)
```

Comment:
beatmapManager?

Reply:
Fixed in https://github.com/ppy/osu/pull/4000/commits/562c9e56fbda6c85dc2ab98402296ec461232eeb